pystac/stac_object.py

```diff
@@ -86,7 +86,7 @@ def get_single_link(
         rel: Union[str, pystac.RelType],
         media_type: Optional[Union[str, pystac.MediaType]] = None,
     ) -> Optional[Link]:
-        """Get single link that match the given ``rel`` and, optionally,
+        """Get single link that matches the given ``rel`` and, optionally,
```

Comment:
```suggestion
        """Get single link that matches the given ``rel`` and
```

Since both are optional, this "optionally" doesn't make much sense.